gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
```

Comment:
I did not understand the need for this change. ( `pravega/pravega` uses `gradle-4.10.2-bin.zip` ) . If we should upgrade gradle wrapper then we should upgrade to the latest version.

Reply:
Hi @shrids 
Annotation processor is available on Gradle version >= **4.6**. Changed it to the **gradle-4.10.2**. 
Lombok version is **1.18.4**, same as used by _pravega/pravega_.